react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export default class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: 0 };
```

Comment:
It is best to name variables and properties so that the name describes the data it holds. So instead of using `isClicked` something like `clickCount` would be a better property name.